lib/starknet_explorer/blockchain/updater_worker.ex

```diff
@@ -1,5 +1,17 @@
 defmodule StarknetExplorer.Blockchain.UpdaterWorker do
+  require Logger
   use GenServer
+  alias StarknetExplorer.TransactionReceipt
+  alias StarknetExplorer.Rpc
+  alias StarknetExplorer.Block
+  alias StarknetExplorer.Gateway
+  alias StarknetExplorer.Repo
+  alias StarknetExplorer.Blockchain.UpdaterWorker
+  defstruct [:network]
+  # Use this parametrized.
+  @network :mainnet
+  @fetch_timer :timer.seconds(2)
```

Comment:
Can we make this value configurable, so we can change it in production without requiring a deploy?